packages/grafana-ui/src/components/Slider/Slider.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { FunctionComponent } from 'react';
+import { Range, createSliderWithTooltip } from 'rc-slider';
+import { css } from 'emotion';
+import { stylesFactory } from '../../themes';
+import { GrafanaTheme } from '@grafana/data';
+import { useTheme } from '../../themes/ThemeContext';
+
+export interface Props {
+  min: number;
+  max: number;
+  orientation: 'horizontal' | 'vertical';
+  // Set current positions of handle(s). If only 1 value supplied, only 1 handle displayed.
```

Comment:
```suggestion
  /** Set current positions of handle(s). If only 1 value supplied, only 1 handle displayed. */
```

Reply:
@ivanahuckova having `//` comments will result in docs not showing in props table. You should use `/** ... */` comments in order for the description to show in storybook docs:
![image](https://user-images.githubusercontent.com/2376619/74909760-4468de80-53b9-11ea-9c7b-bdf72fffce1f.png)


I'm working on styleguide update that covers this: https://github.com/grafana/grafana/pull/22197/files#diff-d7a6487cc6d2e9460188e8527b28839eR204